pos_receipt_replace_user_by_trigram/__manifest__.py

```diff
@@ -11,7 +11,7 @@
     "installable": True,
     "version": "16.0.1.0.0",
     "depends": [
-        "point_of_sale",
+        "pos_hr",
```

Comment:
this should go in a glue module IMO

Reply:
Hi @simahawk, thank you for your in-depth review 👍️

I separate the modules as you suggested.